packages/arcgis-rest-request/src/ArcGISIdentityManager.ts

```diff
@@ -1205,9 +1237,25 @@ export class ArcGISIdentityManager
     };
   }
 
+  /**
+   * Serializes the `ArcGISIdentityManager` instance to a JSON string.
+   *
+   * ```js
+   * // create an ArcGISIdentityManager instance
+   * const serializedString = manager.serialize();
+   * localStorage.setItem("arcgis-identity-manager", serializedString);
+   *
+   * // later, you can retrieve the manager from localStorage
+   * const serializedString = localStorage.getItem("arcgis-identity-manager");
+   * const manager = ArcGISIdentityManager.deserialize(serializedString);
+   * ```
+   *
+   * @returns The serialized JSON string.
+   */
   public serialize() {
     return JSON.stringify(this);
```

Comment:
I know this pre-dates your PR, but this should also probably have been:
```suggestion
    return JSON.stringify(this.toJSON());
```

Reply:
This actually isn't necessary. `JSON.stringify()` calls this internally.

> If the value has a toJSON() method, it's responsible to define what data will be serialized. Instead of the object being serialized, the value returned by the toJSON() method when called will be serialized.
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify